spec/unit/validator_spec.rb

```diff
@@ -894,10 +894,10 @@ def test_swiftpod_with_dot_swift_version(version = '3.1.0')
       end
 
       describe '#swift_version' do
-        it 'defaults to Swift 3.0' do
+        it 'defaults to Swift 3.2' do
```

Comment:
There is a test right above this that contains a warning message starting with `Swift projects uses Swift 3.0 by default` - could that test (and possibly the code that generates that warning) be updated as well?